man/systemd-cgtop.xml

```diff
@@ -170,25 +170,48 @@
       </varlistentry>
 
       <varlistentry>
+        <term><option>-P</option></term>
+
+        <listitem><para>Count only userspace processes instead of all
+        tasks. By default all tasks are counted: each kernel thread
+        and each userspace thread individually. With this setting
+        kernel threads are excluded from the counting and each
+        userspace process only counts as one, regardless how many
+        threads it consists of. This setting may also be toggled at
+        runtime by pressing the <keycap>P</keycap> key. This option
+        may not be combined with
+        <option>-k</option>.</para></listitem>
+      </varlistentry>
+
+      <varlistentry>
         <term><option>-k</option></term>
 
-        <listitem><para>Include kernel threads when counting tasks in
-        control groups. By default, kernel threads are not included in
-        the count. This setting may also be toggled at runtime by
-        pressing the <keycap>k</keycap> key.</para></listitem>
+        <listitem><para>Count only userspace processes and kernel
+        threads instead of all tasks. By default all tasks are
+        counted: each kernel thread and each userspace thread
+        individually. With this setting kernel threads are included in
```

Comment:
this sentence is a bit awkward, how about: `By default every kernel and userspace thread is counted.`

also s/maybe/may be/g